### PR TITLE
Hackathon - Team a

### PR DESCRIPTION
**Pull request by ArmandBriere**

### CODE REVIEW

**Review by CmD0 — APPROVED** [review]
Very Nice, we've had proof it works, I've been following the development all day and i don't see any issue with it.

File: src/functions/exam/main.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    name = name.replace("Examens: ", "")
+    dates = name.split("et")
+    return dates
+
+
+def get_semester() -> str:
+    """Function that returns the current semester"""
+
+    current_year = str(date.today().year)
+    winter = ["01-01-" + current_year, "04-30-" + current_year]
```
Comment (ArmandBriere) [inline src/functions/exam/main.py:34]: winter_start, winter_end = f"01-01-{current_year}", f"04-30-{current_year}"

File: src/functions/exam/main.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    summer = ["01-05-" + current_year, "08-30-" + current_year]
```
Comment (ArmandBriere) [inline src/functions/exam/main.py:35]: summer_start, summer_end = f"01-05-{current_year}", f"08-30-{current_year}"

File: src/functions/exam/main.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    fall = ["01-09-" + current_year, "12-21-" + current_year]
```
Comment (ArmandBriere) [inline src/functions/exam/main.py:36]: fall_start, fall_end = f"01-09-{current_year}", f"12-21-{current_year}"

File: src/functions/exam/main.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    exams = examens_scraper(course, semester)
+    message = ""
+
+    if not exams:
+        return "Je n'ai rien trouvé"
+    if len(exams) == 1:
+        message += "Exam finale: " + exams[0] + "\n"
+    else:
+        message += "Vos exams: \n"
+        message += "Intra: " + exams[0] + "\n"
```
Comment (ArmandBriere) [inline src/functions/exam/main.py:71]: `message += f"Intra: {exams[0]}\n"`

File: src/functions/exam/main.py [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    if not channel_name:
+        return ""
+    return channel_name.lower()
+
+
+def display_exams(course: str, semester: str) -> str:
+    """Function that returns message to be displayed to the channel"""
+
+    exams = examens_scraper(course, semester)
+    message = ""
```
Comment (ArmandBriere) [inline src/functions/exam/main.py:63]: You can delete this and instantiate the var when needed

File: src/functions/exam/main.py [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+    if not exams:
+        return "Je n'ai rien trouvé"
```
Comment (ArmandBriere) [inline src/functions/exam/main.py:66]: `return "Sorry, I didn't find anything"`

**Review by ArmandBriere — COMMENTED** [review]
@wassimkha

**Review by raoufbaz — APPROVED** [review]

**Review by raoufbaz — APPROVED** [review]

**Pull request merged**